### PR TITLE
Allow users to log in and add users listing

### PR DESCRIPTION
**Pull request by Skysplit**
This PR completes **task #1**

### CODE REVIEW

**Comment by Skysplit** [comment]
Sorry, ZSH screwed me up and I didn't add files that were untracked by git repository, and I pushed commit before I realized it :/

**Comment by Skysplit** [comment]
Nevermid, I squashed commits :P

File: config/database.yml [hunk 1]
```diff
@@ -1,25 +1,54 @@
...
+#   http://dev.mysql.com/doc/refman/5.7/en/old-client.html
 #
 default: &default
-  adapter: sqlite3
+  adapter: mysql2
+  encoding: utf8
   pool: 5
-  timeout: 5000
+  username: root
+  password: qweasd123
```
Comment (socjopata) [inline config/database.yml]: Consider using similar technique for providing password, as it is done in the line 54. In the lines 31-49 you can find also a nice overview of how to do that properly.

File: spec/controllers/home_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe HomeController, type: :controller do
+  describe "GET #index" do
+    it "should redirect to login page" do
```
Comment (socjopata) [inline spec/controllers/home_controller_spec.rb]: Consider wrapping these specs with contexts. Like `context "when user is not logged in" do` in the above case

File: spec/helpers/home_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the HomeHelper. For example:
+#
+# describe HomeHelper do
+#   describe "string concat" do
```
Comment (socjopata) [inline spec/helpers/home_helper_spec.rb]: It is usually a good practice not to commit commented out code ;) Also these files are autogenerated. Don't be afraid to delete them if they don't bring any value

**Pull request merged**